pages/api/profiles/[username]/index.js

```diff
@@ -101,22 +101,11 @@ export async function getUserApi(req, res, username) {
     const today = new Date();
     getProfile.events.map((event) => {
       let cleanEvent = JSON.parse(JSON.stringify(event));
-      const dateTimeStyle = {
-        dateStyle: "full",
-        timeStyle: "long",
-      };
       try {
         const start = new Date(event.date.start);
         const end = new Date(event.date.end);
-        cleanEvent.date.startFmt = new Intl.DateTimeFormat(
-          "en-GB",
-          dateTimeStyle
-        ).format(start);
-        cleanEvent.date.endFmt = new Intl.DateTimeFormat(
-          "en-GB",
-          dateTimeStyle
-        ).format(end);
-
+        cleanEvent.date.startFmt = formatDate(start);
+        cleanEvent.date.endFmt = formatDate(end);
```

Comment:
This was `en-GB` before but now will default to `en-US`? 🤔

Reply:
Yes, I have changed it to be en-GB now